### DISCUSSION TITLE
Cómo obtener feedback de nuestras practicas y ejercicios?

### DISCUSSION THREAD

**Discussion by ProDeveloperRoy** [Launch-X-Latam/MisionFrontEnd#15] [Q&A]
Como puedo obtener feedback de los ejercicios practicos que nos dejen para saber si lo estoy aplicando correctamente?

**Comment by JosafatJimenezB** [comment]
Solo comparte tus practicas con la comunidad la cual te ayudara siempre. aquí te podemos dar tips y todo lo que se necesite, saludos!!!

**Reply by ProDeveloperRoy** [reply]
Genial, gracias @JosafatJimenezB

**Reply by JosafatJimenezB** [reply]
denada bro

**Comment by Emilio17MC** [comment]
Puedes compartirlos aquí mismo en Discussions en la categoría de _Show & Tell_

**Comment by OATS74** [comment]
Ayuda no entiendo la mecánica de estudio  ya estoy leyendo los ítem del frontend

**Reply by fedeminatta** [reply]
espera a la clase del miércoles, el comander dijo que ahí va a ir explicando bien

**Comment by doguedogue** [comment]
Aqui te damos feedback, para eso es la comunidad 🚀🚀🚀
